analyzers/tests/SonarAnalyzer.Test/Rules/EmptyMethodTest.cs

```diff
@@ -67,7 +67,7 @@ public class EmptyMethodTest
         public void EmptyMethod_CSharp10() =>
             builderCS.AddPaths("EmptyMethod.CSharp10.cs")
                 .WithOptions(ParseOptionsHelper.FromCSharp10)
-                .Verify();
+                .VerifyNoIssues();  // FN, parameterless struct constructors are not supported
```

Comment:
I'll stop commenting those because they are polluting the review. You get the point:
* I would not justify why you used VerifyNoIssues if all the examples are compliant, there is no need
* I would leave the comments you did for FN, changing the comma to column, as we do for `Noncompliant: reason`

```suggestion
                .VerifyNoIssues();  // FN: parameterless struct constructors are not supported
```